### ISSUE TITLE
Show an error if json failed to load

### ISSUE THREAD

**Issue by lorandcompany** [jahirfiquitiva/Frames#219]
### Is your feature request related to a problem? Please describe.

Someone had an issue where the app just shows "No wallpapers". We gave him the typical troubleshoot instructions. Turns out that the site where the json is contained was blocked by their ISP. This can be communicated better to the user.

### Describe the solution you'd like

- Show something else rather than "No Wallpapers". Maybe "Failed to load wallpapers. Check your internet connection and try again." or something along those lines should work. 
- A toast that says that loading the list failed to load could also suffice. 

Basically anything that indicates that the json failed to load.
